### DISCUSSION TITLE
Icons above players

### DISCUSSION THREAD

**Discussion by Zonharo** [supv-cfx/sublime_core-fivem#28] [Ideas]
1. Displaying such information as a static ID individual for each player and a dynamic one drawn arbitrarily each time the player enters the server. 
2. Addition of icons above the head that show permissions
3. Addition of icons while speaking above the head
4. Addition of death icons above the head
5. Addition of icons of searching by the police
6. After greeting person's name and surname shows if they are not wearing a mask at the time.

As for all the examples, it would be nice if there was a config responsible for the ability to enable and disable these icons each server will be able to customize under itself. Several servers in my country use this and players praise this solution very much. 

**Example photos of this solution**:

<img width="122" alt="Screenshot 2023-05-23 at 14 03 47" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/3fb7c3c6-5dc1-4ee6-9a98-65de9242073e">
<img width="71" alt="Screenshot 2023-05-23 at 13 58 05" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/4d65f421-3319-4d3b-9f69-2818867bb187">
<img width="61" alt="Screenshot 2023-05-23 at 14 01 39" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/d4897d01-7878-49a6-bc2c-344af6eee98c">
<img width="39" alt="Screenshot 2023-05-23 at 13 53 21" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/9d1d90af-21fd-4562-8556-50da90e3bbea">
<img width="101" alt="Screenshot 2023-05-23 at 14 04 55" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/673ef3d4-5479-488d-8c29-e760d4545ea4">
<img width="34" alt="Screenshot 2023-05-23 at 14 08 01" src="https://github.com/SUBLiME-Association/sublime_core/assets/96726807/d575cf44-8c60-4094-8edf-63c37cda7a8e">

**Comment by wewesss** [comment]
Yes, player got static id au increments from database when they create their profile, and each character's got static id too (auto increments from database and we use foreign key to link profile with chars) and yes we go add features you said, can be remove from config 👍🏻
